Wflow/src/routing/surface_local_inertial.jl

```diff
@@ -750,16 +750,19 @@ Compute a stable timestep size for the local inertial approach, based on Bates e
 
 dt = cfl * (Δx / sqrt(g max(h))
 """
-function stable_timestep(model::LocalInertialRiverFlow, flow_length::Vector{Float})
-    dt_min = Inf
+function stable_timestep(
```

Comment:
Note: in testing I found this function scales better on the iGPU than CPU. So even with integrated graphics we can speed up the model 🤔